src/common/state/db_client_table.h

```diff
@@ -65,4 +65,20 @@ typedef struct {
   void *subscribe_context;
 } DBClientTableSubscribeData;
 
+/*
+ * ==== Plasma manager heartbeats ====
+ */
+
+/**
+ * Start sending heartbeats to the plasma_managers channel. Each
+ * heartbeat contains this database client's ID. Heartbeats can be subscribed
+ * to through the plasma_managers channel. Once called, this "retries" the
+ * heartbeat operation forever, every HEARTBEAT_TIMEOUT_MILLISECONDS
+ * milliseconds.
```

Comment:
Hmm the original reason was actually that I wanted to use a `RetryInfo` with 0 retries, and I'd thought that I couldn't do that, but I think I can work around it now.